server/routes/index.js

```diff
@@ -2,8 +2,8 @@ const express = require('express');
 
 const router = express.Router();
 
-router.get('/hi', (req, res) => {
-  res.send('hi');
-});
+const { logOut } = require('../controller');
+
+router.post('/logout', logOut);
 
 module.exports = router;
```

Comment:
please see [this](https://github.com/GSG-G9/beauty-salon/pull/53#discussion_r603544687)